client/src/data/interviews/technical-screening.tsx

```diff
@@ -289,6 +289,56 @@ const theoryQuestions = [
     title: 'Difference between list and array, or between stack and queue.',
     topic: 'Data structures',
   },
+  {
+    id: 'dataTypes',
+    title: 'Understanding data types - from primitives to objects',
+    topic: 'JavaScript Fundamentals',
+  },
+  {
+    id: 'variables',
+    title: 'Variables',
+    topic: 'JavaScript Fundamentals',
+  },
+  {
+    id: 'numbers',
+    title: 'Number & Math methods',
+    topic: 'JavaScript Fundamentals',
+  },
+  {
+    id: 'Strings',
+    title: 'String methods',
+    topic: 'JavaScript Fundamentals',
+  },
+  {
+    id: 'stringTemplates',
+    title: 'String templates',
+    topic: 'JavaScript Fundamentals',
+  },
+  {
+    id: 'ternaryOperator',
```

Comment:
separate question for ternary operator 😄

Reply:
Thanks mate, I updated this question